book/website/reproducible-research/rdm/rdm-dmp.md

```diff
@@ -78,6 +78,17 @@ If you do not select a license others will not be able to reuse your data withou
 
 You can use this [checklist](https://ukdataservice.ac.uk/learning-hub/research-data-management/plan-to-share/checklist/) to see if you have everything covered in your Data Management Plan. 
 
+(rr-rdm-dmp-tools)=
+
+## DMP tools 
+There are several platforms or tools that you can use to set up your Data Management Plan: 
+
+* [DMPonline](https://dmponline.dcc.ac.uk)
+* [DMPtool](https://dmptool.org)
+* [ARGOS](https://argos.openaire.eu/home)
+
```

Comment:
* [RDMO](https://rdmorganiser.github.io)

Reply:
Is that for creating DMPs specifically?

Reply:
yes